addon/utils/assign.ts

```diff
@@ -0,0 +1,11 @@
+import { merge } from '@ember/polyfills';
+
+const assign = /* TODO EmberAssign || */Object.assign || _assign;
```

Comment:
Need to add this back in when this lands: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30424

in: https://www.npmjs.com/package/@types/ember